VitvitskyAkim/HW5/Task5_2.py

```diff
@@ -0,0 +1,11 @@
+n = int(input("How many Fibonacci numbers\
+               to display on the screen?: "))
```

Comment:
look at this implementation:
n = int(input('enter some number from fibonacci list= '))

fibonacci_numbers = [0, 1]

for i in range(2,n):
        fibonacci_numbers.append(fibonacci_numbers[i-1]+fibonacci_numbers[i-2])
print(f'there are some numbers from fibanacci till that you entered {n},\n {fibonacci_numbers}')